web/src/modules/Clusters/ClustersDetails/ClustersDetailsHeader.tsx

```diff
@@ -24,7 +23,7 @@ const ClustersDetailsHeader = ({
             <ListLoading />
           ) : (
             <div
-              className={`${styles.bottomSectionContainer} bgc-white pat-border-t bdrc-medium-purple`}
+              className='rounded overflow-auto bgc-white pat-border-t bdrc-medium-purple'
```

Comment:
```suggestion
              className='rounded overflow-auto bg-white border-t border-medium-purple'
```
I want to get rid of all old library classes. These Tailwind classes should be sufficient replacements. Please check.